### ISSUE TITLE
Fatal crash - NullPointerException - Attempt to invoke virtual method 'void androidx.swiperefreshlayout.widget.SwipeRefreshLayout.setVisibility(int)' on a null object reference

### ISSUE THREAD

**Issue by tingsu** [ram-on/SkyTube#710]
Hi @gzsombor,

The bug was found in the version 2.97. It shares the similar root cause with #704 , but was thrown from different location, i.e., ``DownloadedVideosFragment``.

```
 FATAL EXCEPTION: main
 Process: free.rm.skytube.extra, PID: 15671
 java.lang.NullPointerException: Attempt to invoke virtual method 'void androidx.swiperefreshlayout.widget.SwipeRefreshLayout.setVisibility(int)' on a null object reference
 	at free.rm.skytube.gui.fragments.DownloadedVideosFragment$PopulateDownloadsTask.onPostExecute(DownloadedVideosFragment.java:87)
 	at free.rm.skytube.gui.fragments.DownloadedVideosFragment$PopulateDownloadsTask.onPostExecute(DownloadedVideosFragment.java:71)
 	at android.os.AsyncTask.finish(AsyncTask.java:651)
 	at android.os.AsyncTask.-wrap1(AsyncTask.java)
 	at android.os.AsyncTask$InternalHandler.handleMessage(AsyncTask.java:668)
 	at android.os.Handler.dispatchMessage(Handler.java:102)
 	at android.os.Looper.loop(Looper.java:148)
 	at android.app.ActivityThread.main(ActivityThread.java:5417)
 	at java.lang.reflect.Method.invoke(Native Method)
 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
```

**Comment by gzsombor** [comment]
Thanks, but luckily, I've already fixed in #708 - [here](https://github.com/ram-on/SkyTube/pull/708/files#diff-bea3e0bac19908fdf0357289e32a8a6fR80)

**Comment by tingsu** [comment]
Thanks. Then, I closed this issue.
